app/core/blockchain/transactionmanager.py

```diff
@@ -308,6 +308,9 @@ def econvalidator(self, cur=None):
                     tcode = self.transaction['specific_data']['tokencode']
                     if tcode and tcode != "0" and tcode != "" and tcode != "string":
                         if is_token_valid(self.transaction['specific_data']['tokencode'], cur=cur):
+                            if is_nft(tcode,cur):
+                                logger.error("this is an nft, cant be created further")
+                                return False
                             existing_custodian = get_custodian_from_token(
```

Comment:
@Prannav-k should we also check for amount to be 1 for NFT?

Reply:
yes, this is being checked in the new commit.